frontend/dockerfile/dockerfile_lint_test.go

```diff
@@ -628,6 +629,40 @@ HEALTHCHECK CMD ["/myotherapp"]
 	checkLinterWarnings(t, sb, &lintTestParams{Dockerfile: dockerfile})
 }
 
+func testLegacyKeyValueFormat(t *testing.T, sb integration.Sandbox) {
+	dockerfile := []byte(`
+FROM scratch
+ENV key value
+LABEL key value
```

Comment:
I know there were some concerns about warning about these being potentially _too_ strict.

Wondering if we should have a "more permissive" and "more strict" rule. The most problematic ones are when multiple whitespaces are present;


```dockerfile
ARG key one two
ENV key one two
```

Those look very much alike, but `ARG` means 3 args are defined, but the `ENV` one defines a single `key` env with value `one two`